recipes/viennarna/build.sh

```diff
@@ -10,7 +10,7 @@ fi
 if [ $PY3K -eq 1 ]; then
     VRNA_PYFLAGS="--without-python2"
 else
-    VRNA_PYFLAGS="--without-python"
+    VRNA_PYFLAGS="--with-python2 --without-python"
```

Comment:
I **guess** bioconda is building for py3 only anyway, or?

Reply:
Yup, python 3.8-3.10 at the moment only

Reply:
Ok, so its safe to remove this part from the build script with the next release?!

Reply:
Jep